.github/CODEOWNERS

```diff
@@ -27,6 +27,9 @@
 /components/viz/service/display/starboard/ @youtube/cobalt-starboard-owners
 /ui/ozone/platform/starboard/              @youtube/cobalt-starboard-owners
 
+/starboard/tvos              @youtube/cobalt-tvos
```

Comment:
QQ: What about //starboard/darwin? Or will that disappear?

Reply:
That one will disappear, hopefully Really Soon (TM). It's a C25 artifact and most of the contents are outdated or will be migrated to `//starboard/tvos`.